plugins/node/opentelemetry-instrumentation-graphql/src/types.ts

```diff
@@ -57,6 +59,18 @@ export interface GraphQLInstrumentationConfig extends InstrumentationConfig {
    */
   mergeItems?: boolean;
 
+  /**
+   * Don't create spans for the execution of the default resolver on object peroperties.
+   *
+   * When a resolver function is not defined on the schema for a field, graphql will
+   * use the default resolver which just looks for a property with that name on the object.
+   * If the property is not a function, it's not very interesting to trace.
+   * This option can reduce noise and number of spans created.
+   *
+   * @default false
+   */
+  ignoreTrivialResolveSpans?: boolean;
```

Comment:
NIt, move the param between `depth` and `mergeItems` to keep them in alphabetical order